src/content/configuration/stats.md

```diff
@@ -115,21 +120,21 @@ module.exports = {
     errorDetails: true,
 
     // Exclude assets from being displayed in stats
-    // This can be done with a String, a RegExp, a Function getting the assets name
-    // and returning a boolean or an Array of the above.
-    // Possible values: String | RegExp | (assetName) => boolean | [String, RegExp, (assetName) => boolean]
+    // This can be done with a String, a RegExp, a Function that receives the asset’s name
+    // and returns a Boolean, or an Array of the above 3 data types.
+    // Possible values: String | RegExp | (assetName) => Boolean | [String, RegExp, (assetName) => Boolean]
```

Comment:
A writers guide was created, please change these notations according to it: https://webpack.js.org/contribute/writers-guide/

Reply:
I’m sorry, I’m not smart enough to understand which part of the writer’s guide you mean. Would you mind being more specific?

Reply:
Please have a look at this https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types